### PR TITLE
Text changes for warning_modal

### PR DESCRIPTION
**Pull request by thess**
Ref: [How to escape backtick in markdown](https://meta.stackexchange.com/questions/82718/how-do-i-escape-a-backtick-within-in-line-code-in-markdown)

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -2,16 +2,18 @@ en:
...
+      `` `` ``
       ```
+      `` `` ``
       multiple
       lines
+      `` `` ``
       ```
-      ````
-    do_not_show_again: don’t show this message again
+      `` `` ``
```
Comment (davidtaylorhq) [inline locales/en.yml:16]: What is the purpose of this change? When I put the updated version into Discourse, it renders very strangely:

Screenshot 2021-09-13 at 12 11 47

**Comment by thess** [comment]
This change was the result of handling the problems brought up in the Discourse forum by _tmomas_ (one of our moderators). An alternate solution to his issues would be great 😃 

https://meta.discourse.org/t/unformatted-code-detector-theme-component/112773/43

I may have mis-implemented the base version posted here. Our version contained additional data as mentioned in the discussion.

**Comment by davidtaylorhq** [comment]
I understand now, thanks. I think the root cause here was a bug in Discourse core - I've posted about it [here](https://meta.discourse.org/t/unformatted-code-detector-theme-component/112773/48?u=david) on Meta.

I think some parts of this PR are still useful though. Please can you remove the backtick escaping I commented on, and then we can merge the other copy changes 😄

**Review by davidtaylorhq — APPROVED** [review]
